packages/testing/README.md

```diff
@@ -0,0 +1,108 @@
+<p align="center">
+  <a href="https://clerk.com?utm_source=github&utm_medium=clerk_testing" target="_blank" rel="noopener noreferrer">
+    <picture>
+      <source media="(prefers-color-scheme: dark)" srcset="https://images.clerk.com/static/logo-dark-mode-400x400.png">
+      <img src="https://images.clerk.com/static/logo-light-mode-400x400.png" height="64">
+    </picture>
+  </a>
+  <br /> 
+</p>
+
+# @clerk/testing
+
+<div align="center">
+
+[![Chat on Discord](https://img.shields.io/discord/856971667393609759.svg?logo=discord)](https://clerk.com/discord)
+[![Clerk documentation](https://img.shields.io/badge/documentation-clerk-green.svg)](https://clerk.com/docs?utm_source=github&utm_medium=clerk_testing)
+[![Follow on Twitter](https://img.shields.io/twitter/follow/ClerkDev?style=social)](https://twitter.com/intent/follow?screen_name=ClerkDev)
+
+[Changelog](https://github.com/clerk/javascript/blob/main/packages/testing/CHANGELOG.md)
+·
+[Report a Bug](https://github.com/clerk/javascript/issues/new?assignees=&labels=needs-triage&projects=&template=BUG_REPORT.yml)
+·
+[Request a Feature](https://feedback.clerk.com/roadmap)
+·
+[Ask a Question](https://github.com/clerk/javascript/discussions)
+
+</div>
+
+---
+
+## Overview
+
+This package provides utilities for testing Clerk applications.
+
+It currently supports the following testing frameworks:
+
+- [Playwright](https://playwright.dev/), a Node.js library to automate browsers and web pages.
+
+## Getting started
+
+### Prerequisites
+
+- Node.js `>=18.17.0` or later
+- Playwright v1+
+
+### Installation
+
+```shell
+npm install @clerk/testing --save-dev
+```
+
+## Usage
+
+### Playwright
+
+On your global setup file for Playwright, you must use the `clerkSetup` function to set up Clerk for your tests.
```

Comment:
🔧 We should also say that the first step is placing your secret key in the `.env.local` or something

Reply:
I'm actually using the following text in the high-level docs - you might want to use the same:

> In a [global setup file](https://playwright.dev/docs/test-global-setup-teardown) for Playwright, call the `clerkSetup` function:

Reply:
> Firstly, add your Clerk keys (Publishable Key and Secret Key) to your environment variables file (e.g. `.env.local` or `.env.`). 
You can find these keys in your Clerk Dashboard.

What do you think?

Reply:
yep, makes sense to mention publishable keys as well, but we should mention the actual env vars they have to use (`CLERK_SECRET_KEY` etc.)